fed/mosaic-omnetpp/src/main/java/org/eclipse/mosaic/fed/omnetpp/ambassador/OmnetppAmbassador.java

```diff
@@ -43,12 +46,15 @@ public OmnetppAmbassador(AmbassadorParameter ambassadorParameter) {
 
     @Nonnull
     @Override
-    public FederateExecutor createFederateExecutor(String host, int port, OperatingSystem os) {
+    public FederateExecutor createFederateExecutor(String host, int port, OperatingSystem os) throws FileNotFoundException {
+        String omnetppConfigFileName = ObjectUtils.defaultIfNull(config.federateConfigurationFile, "omnetpp.ini"); // default // this one is required
+        if(!Files.exists(Paths.get(this.ambassadorParameter.configuration.getParent(), omnetppConfigFileName))) {
+            throw new FileNotFoundException(omnetppConfigFileName + " missing");
```

Comment:
This would make sense for `Ns3Ambassador` as well, wouldn't it?

Reply:
You could also use:

```java
Validate.isTrue(
   Files.exists(Paths.get(ambassadorParameter.configuration.getParent(), omnetppConfigFileName),
   "OMNet++ configuration file ' {}' missing.", omnetppConfigFileName
);
```
which throws an `IllegalArgumentException` in case it validates to false.